proposals/Exceptions.md

```diff
@@ -444,8 +159,9 @@ Try blocks, like control-flow blocks, have a _block type_. The block type of a
 try block defines the values yielded by the evaluation the try block when either
 no exception is thrown, or the exception is successfully caught by the catch
 block.
+
 Try blocks, like control-flow blocks, have a _block type_. The block type of a
-try block defines the values yielded by the evaluation the try block when either
+try block defines the values yielded by the evaluating the try block when either
```

Comment:
Should probably be "...by the evaluation of the try block..."

Reply:
Yeah I fixed that in the second (duplicated) paragraph